### ISSUE TITLE
Edit Question:  /metadata-and-referencing/structure-granularity-and-referencing/pids/subentities

### ISSUE THREAD

**Issue by jmfrenzel** [FDM-UARuhr/rdmo-catalog-uaruhr#105]
**Describe the request**  
User feedback to change German question text. 

https://fdm-uaruhr.github.io/rdmo-catalog-uaruhr/questions/ua_changes.html

**Propose what exactly should be changed**  
As suggested below, please choose from the list in Additional context and provide the updated values.

**Original setting or content**   
What is the current content or setting e.g. 

*question text en*  
"original text"

**Proposed setting or content**   
What should it look like in the new version, e.g. 

*question text en*  
"new text"

**Additional context**

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  
- [ ] uri
- [ ] key
- [ ] path
- [ ] questionset
  - [ ] position in set
- [ ] question text (en and ger)
- [ ] help text (en and ger)
- [ ] answer
  - [ ] widget type
  - [ ] value type
  - [ ] max/min, step and unit
- [ ] optionset
- [ ] condition
- [ ] attribute

**Comment by ClaraRosalie** [comment]
created PR in upstream https://github.com/rdmorganiser/rdmo-catalog/pull/120

**Comment by jmfrenzel** [comment]
Integrate upstream changes via cherry-pick and update  uaruhr catalogs and tooling (./toools/all_questions.xml)
